### DISCUSSION TITLE
[Question] Plans for wlroots support and extras?

### DISCUSSION THREAD

**Discussion by A-Cloud-Ninja** [pinnacle-comp/pinnacle#19] [General]
Are there any plans for this to support wlroots extensions like layer-shell? I notice you have a widget system planned, so I'm not sure if you intend on that being entirely separately implemented to replace it or alongside of.

Otherwise I'm super interested and watching every commit :eyes:

**Comment by Ottatop** [comment]
The plan is to support just layer-shell as it's the only wlroots protocol that Smithay has implemented at the moment. The widget system, while still a *really* long way off, will be built alongside it.

Edit: Ok scratch the part about *only* layer-shell, I've discovered that Smithay re-exports wlr and other protocols. I just have to implement them myself, so stuff like wlr-screencopy and wlr-output-management are now on the table!
